storageservice/modules/files/file_client.bal

```diff
@@ -253,12 +250,12 @@ public client class FileClient {
     # + azureFileName - Name of the file in azure
     # + azureDirectoryPath - Path of the azure directory
     # + return - If success, returns true.  Else returns error
-    @display {label: "Write content to file"}
-    remote function putRange(@display {label: "File share name"} string fileShareName,
-                             @display {label: "Local file path"} string localFilePath, 
-                             @display {label: "Name of the file in azure"} string azureFileName, 
-                             @display {label: "Azure directory path"} string? azureDirectoryPath = ()) 
-                             returns @tainted @display {label: "Status"} boolean|error {
+    @display {label: "Write Content To Azure File"}
+    isolated remote function putRange(@display {label: "File Share Same"} string fileShareName,
```

Comment:
File Share Name? However, what is meant by `File Share Name`?

Reply:
There is a few places we have done same mistake.

Reply:
File Shares are the file containers of Azure File Service.

Reply:
+1